tracer/build/_build/Build.GitHub.cs

```diff
@@ -33,9 +33,18 @@ partial class Build
     [Parameter("A GitHub token (for use in GitHub Actions)", Name = "GITHUB_TOKEN")]
     readonly string GitHubToken;
 
+    [Parameter("Git repository name", Name = "GITHUB_REPOSITORY_NAME", List = false)]
+    readonly string GitHubRepositoryName = "dd-trace-dotnet";
+
     [Parameter("An Azure Devops PAT (for use in GitHub Actions)", Name = "AZURE_DEVOPS_TOKEN")]
     readonly string AzureDevopsToken;
 
+    [Parameter("Azure Devops pipeline id", Name = "AZURE_DEVOPS_PIPELINE_ID", List = false)]
+    readonly int AzureDevopsConsolidatePipelineId = 54;
+
+    [Parameter("Azure Devops project id", Name = "AZURE_DEVOPS_PROJECT_ID", List = false)]
+    readonly Guid AzureDevopsProjectId = Guid.Parse("a51c4863-3eb4-4c5d-878a-58b41a049e4e");
```

Comment:
I assume you've tested that you can set a guid from the command line? 🤔

Reply:
Yes, sure :)
Nuke does awesome work in converting variables.